packages/helm-charts/load-test/templates/load-test.statefulset.yaml

```diff
@@ -109,14 +109,12 @@ spec:
           ACCOUNT_ADDRESSES=$(cat /root/.celo/address | tr '\n' ',')
           ACCOUNT_ADDRESSES=${ACCOUNT_ADDRESSES::-1}
 
+{{ include  "common.geth-http-ws-flags" (dict "Values" $.Values "rpc_apis" "eth,web3,debug,admin,personal,net" "ws_port" "8545" "listen_address" "0.0.0.0") | indent 10 }}
+
           exec geth \
           --datadir /root/.celo \
           --ipcpath=geth.ipc \
           --nousb \
-          --rpc \
```

Comment:
`--http` ?

Reply:
The `--http` flag is included in the include `common.geth-http-ws-flags`.